client/src/components/TreeDropdownField/TreeDropdownField.js

```diff
@@ -405,10 +423,15 @@ class TreeDropdownField extends Component {
    */
   renderMenu(renderMenuOptions) {
     // Build root node
-    const tree = this.getVisibleTree() || {};
+    const visibleTree = this.getVisibleTree();
```

Comment:
are you talking about this line? https://github.com/silverstripe/silverstripe-admin/pull/67/files#diff-3eeee0f0f28cfe1453d1922e6333dc5fR129

I'm not sure I understand why the comment was placed here
root appears when `showRootOption` is true and you're on the root directory.